cli/validate_cmd.go

```diff
@@ -113,8 +113,7 @@ walkLoop:
 
 		pth := walker.Path()
 
-		fi, err := os.Stat(pth)
-		if !fi.IsDir() {
+		if fi, err := os.Stat(pth); err != nil || !fi.IsDir() {
```

Comment:
We wouldn't want to check [`os.IsPermission(err)`](https://godoc.org/os#IsPermission) or anything like that, would we?